File: striver/S3L1/max_staright_ones.java
package S3L1;

public class max_staright_ones {
    public int findMaxConsecutiveOnes(int[] nums) {
        int c=0;
        int mc=0;
        for(int num:nums){
            if(num==1){
                c++;
            }else {
                mc = Math.max(mc,c);
                c=0;
            }
        }
        mc = Math.max(mc,c);
        return mc;
    }
}

File: striver/S3L1/max_staright_ones.py
class Solution(object):
    def findMaxConsecutiveOnes(self, nums):
        c=0
        mc=0
        for num in nums:
            if num:
                c+=1
            else:
                mc=max(mc,c)
                c=0
        mc=max(mc,c)
        return mc